README.md

```diff
@@ -1,23 +1,56 @@
 # The Sixie Opinionated Django Cookiecutter
 
-This is a work in progress, but it is heavily inspired by the work
+This is a work in progress project for jumpstarting production-ready Django projects. It is heavily inspired by the work
 of [cookiecutter-django](https://github.com/pydanny/cookiecutter-django)
 and Daniel Roy Greenfeld. It also borrows heavily from the ideas in the
 PyPA [warehouse](https://github.com/pypa/warehouse) as I feel they have 
 made an amazing developer onboarding experience for a very complicated
 project.
 
-Creating a new project using this repo:
+Clone this repo:
```

Comment:
**suggestion** I changed the procedure to remove the need of cloning this repository (as cookiecutter already manage it). See my PR and comment there.